wire-runtime/src/commonMain/kotlin/com/squareup/wire/ProtoAdapter.kt

```diff
@@ -222,7 +231,7 @@ internal inline fun <E> ProtoAdapter<E>.commonCreatePacked(): ProtoAdapter<List<
 
 internal class PackedProtoAdapter<E>(
   private val originalAdapter: ProtoAdapter<E>
-) : ProtoAdapter<List<E>>(FieldEncoding.LENGTH_DELIMITED, List::class) {
+) : ProtoAdapter<List<E>>(FieldEncoding.LENGTH_DELIMITED, List::class, null) {
```

Comment:
@swankjesse We could potentially fill all those `null`s with some `"square.github.io/wire/scalar"` and change the field form being nullable to non-nullable?

Reply:
I think I prefer it as-is? There is no type URL for scalar types and it isn’t possible to encode them here